test/integration/volumescheduling/util.go

```diff
@@ -54,10 +54,10 @@ type testContext struct {
 	cancelFn context.CancelFunc
 }
 
-// initTestMaster initializes a test environment and creates a master with default
+// InitTestAPIServer initializes a test environment and creates a master with default
```

Comment:
making this method public seems unrelated to the goal of this PR

Reply:
I didn't see the case when I replaced it, so let me fix that.

Reply:
Updated.